crates/sui-keys/src/keystore.rs

```diff
@@ -270,6 +394,10 @@ impl AccountKeystore for InMemKeystore {
             None => Err(anyhow!("Cannot find key for address: [{address}]")),
         }
     }
+
+    fn create_alias_if_not_exists(&self, _alias: &Option<String>) -> Result<String, anyhow::Error> {
```

Comment:
Yes, I've been torn on it. Initially I had aliases for the `InMemKeystore` but it kinda does not make sense to have one, I think. I'll fix this, thank you.

Reply:
`&dyn AccountKeystore` might not work because `sign_hashed` takes a generic type `T`, and the compiler complains. I'll find a workaround.
<img width="1703" alt="image" src="https://github.com/MystenLabs/sui/assets/135084671/4a4a5e37-db8b-46ed-a3ca-7c6691f8a7a1">